tests/accelerators/legacy/test_ddp.py

```diff
@@ -96,7 +97,7 @@ def test_cli(tmpdir):
 # END: test_cli ddp test
 
 
-@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+@pytest.mark.skipif(**_SKIPIF_NO_GPUS)
```

Comment:
it's not equivalent to saying <2 gpus

Reply:
it is, note difference `_skipif_no_gou` and
 `_skipif_no_gouS`

Reply:
This is a bit of a problem with natural language here. 
May I suggest a decorator with argument? For example:
`@device_required(gpus=2)`
or
`@gpu_required(2)`
or something like that?

Reply:
well yes, my main motivation was king of simple chaining conditions to be easier to read it...

Reply:
well, @awaelchli but for your two you in fact mean `min_gpus=2`

Reply:
yes, that should be fine. If 2 are required we are also happy with >=2. We could name the positional arg min_gpus

Reply:
mind comment on https://github.com/PyTorchLightning/pytorch-lightning/pull/5920#issuecomment-781435819